src/core/psi_container.jl

```diff
@@ -221,6 +221,7 @@ model_initial_time(psi_container::PSIContainer) = psi_container.initial_time
 #Internal Variables, Constraints and Parameters accessors
 get_variables(psi_container::PSIContainer) = psi_container.variables
 get_constraints(psi_container::PSIContainer) = psi_container.constraints
+get_constraints(psi_container::PSIContainer, name::Symbol) = psi_container.constraints[name]
```

Comment:
Isn't this the same as `get_constraint` ?

Reply:
indeed... I didn't see that.